### PR TITLE
Created reference page for NuGet CLI's search command

### PR DESCRIPTION
**Pull request by advay26**

### CODE REVIEW

**Comment by anangaur** [comment]
Docs Build status updates of commit _[6da1fc4](https://github.com/advay26/docs.microsoft.com-nuget/commits/6da1fc4f99546d34df1ce266aa0b109071c800b6)_: 

:clock10: **Pending**: waiting for processors ([10](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 10 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

**Comment by anangaur** [comment]
Docs Build status updates of commit _[6da1fc4](https://github.com/advay26/docs.microsoft.com-nuget/commits/6da1fc4f99546d34df1ce266aa0b109071c800b6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/cli-reference/cli-ref-search.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/reference/cli-reference/cli-ref-search.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-search?branch=pr-en-us-2093) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C7%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202007072151593146-2093%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=HvAAHdqK8RNkwcAgYcLIr5HLYf3TN%2FIOiMAn5zHdh68%3D&st=2020-07-07T21%3A47%3A30Z&se=2020-08-07T21%3A52%3A30Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by anangaur** [comment]
Docs Build status updates of commit _[b19594e](https://github.com/advay26/docs.microsoft.com-nuget/commits/b19594edf5516f6f18d50dbacd8f58d47c783edc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-search?branch=pr-en-us-2093) |
[docs/reference/cli-reference/cli-ref-search.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/reference/cli-reference/cli-ref-search.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-search?branch=pr-en-us-2093) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C7%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202007072158475324-2093%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=T6TBZAioVNba6gRYF77zqyKUUvXRsdJhCgMTN8eIUAo%3D&st=2020-07-07T21%3A54%3A17Z&se=2020-08-07T21%3A59%3A17Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/reference/cli-reference/cli-ref-search.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+description: Reference for the nuget.exe search command
+author: advay26
+ms.author: t-adtand
+ms.date: 07/07/2020
+ms.topic: reference
+---
+
+# search command (NuGet CLI)
+
+**Applies to:** package consumption &bullet; **Supported versions:** all
```
Comment (zivkan) [inline docs/reference/cli-reference/cli-ref-search.md:12]: `nuget search` is not available in all versions of nuget.exe.

Comment (advay26) [reply docs/reference/cli-reference/cli-ref-search.md:12]: Yes, I thought as much, but wasn't sure what the right answer would be. Do you happen to know which versions would support `search`?

**Comment by anangaur** [comment]
Docs Build status updates of commit _[e8315ed](https://github.com/advay26/docs.microsoft.com-nuget/commits/e8315ed598b7eff7fd1b6cc53775d49395a72506)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-search?branch=pr-en-us-2093) |
[docs/reference/cli-reference/cli-ref-search.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/reference/cli-reference/cli-ref-search.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-search?branch=pr-en-us-2093) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C7%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202007072238479897-2093%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=M1i0MmQGewuolZBqrdsWK5bKX9NRYBTNb%2Fj9jShWWAc%3D&st=2020-07-07T22%3A34%3A18Z&se=2020-08-07T22%3A39%3A18Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/reference/cli-reference/cli-ref-search.md (hunk 1 above)
Comment (zivkan) [reply docs/reference/cli-reference/cli-ref-search.md]: NuGet.Client's dev branch is in the process of being rebranded to 5.8, but it depends on when the feature was merged into client's dev branch.

Comment (advay26) [reply docs/reference/cli-reference/cli-ref-search.md]: I've changed it to TODO for now, and I'll update it based on when the command is merged with dev.

File: docs/reference/cli-reference/cli-ref-search.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+```
+nuget search logging -Take 5
+```
+Search for *JSON*-related packages, including pre-release versions, from specified source/feed:
+```
+nuget search JSON -PreRelease -Source "https://api.nuget.org/v3/index.json"
+```
+Search for *JSON*-related packages from multiple sources/feeds:
+```
+nuget search JSON -Source "https://api.nuget.org/v3/index.json" -Source "https://other-feed-url-goes-here"
```
Comment (zkat) [inline docs/reference/cli-reference/cli-ref-search.md:64]: question re the code: Did we actually test this? I didn't know this would work! That's cool!

Comment (advay26) [reply docs/reference/cli-reference/cli-ref-search.md:64]: Yup, this works!

**Comment by anangaur** [comment]
Docs Build status updates of commit _[075b03d](https://github.com/advay26/docs.microsoft.com-nuget/commits/075b03dffc70b8e8c17ad09c7e05fa57cd16cf33)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-search?branch=pr-en-us-2093) |
[docs/reference/cli-reference/cli-ref-search.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/reference/cli-reference/cli-ref-search.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-search?branch=pr-en-us-2093) |
[docs/reference/nuget-exe-CLI-Reference.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/reference/nuget-exe-CLI-Reference.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference?branch=pr-en-us-2093) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C8%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202007082207478536-2093%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=FDxiH76mMxl%2BMETEy0yw%2BYSu%2Bt6IZHsLTp9IPRwn7%2F8%3D&st=2020-07-08T22%3A03%3A19Z&se=2020-08-08T22%3A08%3A19Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by anangaur** [comment]
Docs Build status updates of commit _[89bbb7c](https://github.com/advay26/docs.microsoft.com-nuget/commits/89bbb7c52b9f91feff41690695d882f47daaf2ce)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-search?branch=pr-en-us-2093) |
[docs/reference/cli-reference/cli-ref-search.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/reference/cli-reference/cli-ref-search.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-search?branch=pr-en-us-2093) |
[docs/reference/nuget-exe-CLI-Reference.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/reference/nuget-exe-CLI-Reference.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference?branch=pr-en-us-2093) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C17%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008171739156316-2093%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=RvENiGipQEorFPCgYHlEjbZ8Ih8lMLTMLMYjJheEob4%3D&st=2020-08-17T17%3A34%3A56Z&se=2020-09-17T17%3A39%3A56Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by anangaur** [comment]
Docs Build status updates of commit _[2621bf2](https://github.com/advay26/docs.microsoft.com-nuget/commits/2621bf2d2846d0cbe46bd0fb178a0abea51d138b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/TOC.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-search?branch=pr-en-us-2093) |
[docs/reference/cli-reference/cli-ref-search.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/reference/cli-reference/cli-ref-search.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-search?branch=pr-en-us-2093) |
[docs/reference/nuget-exe-CLI-Reference.md](https://github.com/advay26/docs.microsoft.com-nuget/blob/master/docs/reference/nuget-exe-CLI-Reference.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference?branch=pr-en-us-2093) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C17%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008171740592167-2093%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=awmFHUdo5rm5x99T699Hp91WgPS7jpdanTHb8Dgn9F8%3D&st=2020-08-17T17%3A36%3A37Z&se=2020-09-17T17%3A41%3A37Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by anangaur** [comment]
Is this supported on dotnet CLI or only nuget.exe for now?

**Comment by JonDouglas** [comment]
@anangaur It's nuget CLI only. We have to backlog a dotnet CLI experience. I'll make a dotnet CLI epic to cover it shortly.

File: docs/reference/cli-reference/cli-ref-search.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+description: Reference for the nuget.exe search command
+author: advay26
+ms.author: t-adtand
+ms.date: 08/17/2020
+ms.topic: reference
+---
+
+# search command (NuGet CLI)
+
+**Applies to:** package consumption &bullet; **Supported versions:** 5.8+
```
Comment (anangaur) [inline docs/reference/cli-reference/cli-ref-search.md:12]: We would need to make it clear that it is available with nuget.exe only

Comment (JonDouglas) [reply docs/reference/cli-reference/cli-ref-search.md:12]: How might we make that more clear? From my review, this is scoped to `NuGet CLI` sections & pages only.

Comment (anangaur) [reply docs/reference/cli-reference/cli-ref-search.md:12]: my bad then... I didn't realize this page was NuGet.exe only.

**Pull request merged**